scripts/create-release-pr.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+if [ -z "${1:-}" ]
+then
+  echo "Error: Missing required positional argument: patch|minor|major"
+  exit 1
+fi
+
+function push_version_bump() {
+  local newVersion
+  newVersion=$(node -p 'require("./lerna.json").version')
+
+  local branchName
+  branchName="release-v${newVersion}"
+
+  git checkout -b "${branchName}"
+  git add .
+  git commit -m "${branchName}" || true
```

Comment:
Nit: We do still typically use the version number in the actual commit message
```suggestion
  git commit -m "${newVersion}" || true
```

Reply:
9e62db9